src/models/Plugins.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-17.
+ */
+
```

Comment:
@Diachk header 
// TS2019-DONE

Reply:
What does this mean in the rest client ?